### ISSUE TITLE
Missing cards in the "Today" column of the "Upcoming cards" section

### ISSUE THREAD

**Issue by ig-perez** [nextcloud/deck#3023]
First of all, thanks a lot for maintaining this app!. Also I have searched for duplicate issues and I couldn't find one. Forgive me if I missed some.

**Is your feature request related to a problem? Please describe.**
First thing in the morning I go to the "Upcoming cards" to plan my day. For this I check the "Today" column which should include cards with deadline set to today and also cards with a future deadline but I've already started working on (meaning they are in the "Doing" list of its respective board). I can not see cards with future deadline that are inside "Doing" lists.

**Describe the solution you'd like**
I'd like to see both kind of cards in the "Today" column of the "Upcoming cards" section. Otherwise the plan for today might miss some activities or be unrealistic from the workload perspective.

**Describe alternatives you've considered**
Review all boards and see which cards are on the "Doing" list and take those into account when planning the day. Sadly, I believe this defeats the purpose of the "Today" column of the "Upcoming cards" section.

**Additional context**
I understand that board list names are not standardized, but it is pretty common to have "To Do", "Doing", and "Done". Perhaps a list can be managed to know which list names should be considered when including on-going cards to the "Today" column. For example `ongoing = ["doing", "in progress"]`

![image](https://user-images.githubusercontent.com/33359427/116090806-de858000-a669-11eb-94d5-b1c295126c73.png)

**Comment by stefan-niedermann** [comment]
As far as i know, the logic is currently:
- The card with an assigned due date is only shown if it is assigned to you as an user
- If the card is on a board which is not shared with anybody else, it will also show if it is not assigned to you

Maybe the cards which you are missing aren't assigned to your user yet because you assign only cards which are in `Doing`? If the board is shared, then you won't be able to see it.

@jakobroehrl or @juliushaertl can you confirm this with a :+1: ?

**Comment by ig-perez** [comment]
Thanks @stefan-niedermann . My boards are for my own usage and not shared. I assign all tasks to myself and the ones in the "To Do" or "Doing" lists have a due date. For example this sample card:
![image](https://user-images.githubusercontent.com/33359427/116103471-3bd2fe80-a675-11eb-8a66-6878687d5c67.png)

Sadly it is in the "Later" column:
![image](https://user-images.githubusercontent.com/33359427/116103621-602edb00-a675-11eb-8e29-abfdf981fe60.png)

**Comment by stefan-niedermann** [comment]
Ah, now i got the issue. Yeah, the upcoming cards view are orientated at the due date.

**Comment by stefan-niedermann** [comment]
My 2 ct: This is valid for your workflow, but might not necessarily match every use case and be actually wrong for other kinds of usages: For example [not everyone is using cards and lists in a "Kanban-like" flow](https://help.nextcloud.com/uploads/default/original/3X/6/8/6877e0108faf3f30389a04da4fceaf6ac7eba1bc.jpeg).

**Comment by ig-perez** [comment]
Yeah, totally agree. I also have those type of boards and this feature request won't apply there because the lack of the "Doing" label:
![image](https://user-images.githubusercontent.com/33359427/116117996-81e28f00-a682-11eb-8a49-ab4c93b2f7ec.png)

Hopefully this request is up-voted since I guess it is a common need among those managing multiple and parallel projects. Actually, and depending on your setup, this might apply to all since we are constantly "managing" multiple projects in life: at school, work, errands, research, hobbies, etc.

**Comment by leegarrett** [comment]
+1 for this. Setting deadlines for tasks is tedious for the upcoming cards section to work and is to static. Sometimes I have 5 tasks for today, but only get three done. I don't want to have to update the deadline of those two remaining tasks, and any follow-up tasks for it to fit again. The information already there by the card being in the "today" list.

I would be great if I could tag one list per board as "today", so I can see that in the overview. Or you could require the columns to be named "today", "tomorrow", "this week", "later" to work. Btw, no idea when a card should be listed in "no due", I've never seen cards listed there under any circumstance.
